dot/state/tries.go

```diff
@@ -30,11 +29,12 @@ var (
 	})
 )
 
+const MaxInMemoryTries = 100
```

Comment:
How this was defined? Should we maybe add some description for this

Reply:
This PR is still WIP, I want to use the same number used in substrate but I have to do a little research to find it. We don't use to review draft PRs because the code can change. Anyways, thanks for the comment.